src/Mono.Android/Java.Interop/JavaObjectExtensions.cs

```diff
@@ -113,11 +109,7 @@ internal static IJavaObject JavaCast (IJavaObject instance, Type resultType)
 				return CastClass (instance, resultType);
 			}
 			else if (resultType.IsInterface) {
-				Type invokerType = GetHelperType (resultType, "Invoker");
-				if (invokerType == null)
-					throw new ArgumentException ("Unable to get Invoker for interface '" + resultType.FullName + "'.", "resultType");
-				var getObject = invokerType.GetMethod ("GetObject", new[]{typeof (IntPtr), typeof (JniHandleOwnership)});
-				return (IJavaObject) getObject.Invoke (null, new object[]{instance.Handle, JniHandleOwnership.DoNotTransfer});
+				return Java.Lang.Object.GetObject (instance.Handle, JniHandleOwnership.DoNotTransfer);
```

Comment:
I'm less certain about this change: `Java.Lang.Object.GetObject()` will return any previously created wrapper for the given type handle, which need not be of type `resultType`.

This should probably be:

```csharp
return Java.Lang.Object.GetObject (instance.Handle, JniHandleOwnership.DoNotTransfer, resultType);
```